nin/backend/fontgen.js

```diff
@@ -0,0 +1,44 @@
+const fs = require('fs-promise');
+const glob = require('glob');
+const path = require('path');
+
+
+async function fontGen(pathPrefix) {
+
+  let resolver;
+  let promise = new Promise(resolve => {
```

Comment:
I think this is a nice tradeoff to avoid multi-level nesting. One alternative is switching out `glob` for `globby`, which gives us a nice promise based api instead of the callback to the glob function.